apps/web/src/app/providers.tsx

```diff
@@ -33,19 +33,22 @@ export function IdentifyUser({
 }) {
   const posthog = usePostHog()
 
+  const email = user?.email
   useEffect(() => {
+    if (!posthog || !email) return
+
     try {
-      if (user && !user.email.match(/@latitude\.so/)) {
-        posthog?.identify(user.email, {
-          email: user.email,
-        })
-        posthog?.group('workspace', String(workspace.id))
-        posthog?.startSessionRecording()
-      }
+      const isStaff = !!email.match(/@latitude\.so/)
```

Comment:
```suggestion
      const isStaff = !!email.match(/@latitude\.so$/)
```

this avoids matching some malicious emails such as `@latitude.so.attacker.domain`